packages/multiple-did-identities/src/__tests__/did-identity-service.spec.ts

```diff
@@ -0,0 +1,263 @@
+import { EventEmitter } from 'eventemitter3';
+import { DIDIdentityService } from '../lib/DIDIdentityService'; // adjust the path as necessary
+import { DidMethodFactory } from '../did-methods/DidMethodFactory';
+import { DIDKeyPair } from '../did-methods/IDidMethod';
+import { DidRepository } from '../repository/DidRepository';
+import { DidEventChannel } from '../utils/DidEventChannel';
+import { ServiceResponseStatus } from '@adorsys-gis/status-service';
+
+describe('DIDIdentityService', () => {
+  let service: DIDIdentityService;
+  let mockEventBus: EventEmitter;
+
+  beforeEach(() => {
+    mockEventBus = new EventEmitter();
+    service = new DIDIdentityService(mockEventBus);
+
+    // Spy on static method DidMetDIDKeyPairhodFactory.generateDid
+    jest.spyOn(DidMethodFactory, 'generateDid');
+
+    // Spy on DidRepository prototype methods
+    jest.spyOn(DidRepository.prototype, 'createDidId');
+    jest.spyOn(DidRepository.prototype, 'deleteDidId');
+    jest.spyOn(DidRepository.prototype, 'getADidId');
+    jest.spyOn(DidRepository.prototype, 'getAllDidIds');
```

Comment:
You're already including `"fake-indexeddb": "^6.0.0"`. Can you update your tests to use it directly? That will allow covering the actual implementation of DidRepository. Tip: Use the following command to monitor the coverage:
```bash
npm run test --workspace packages/multiple-did-identities -- --coverage
```